### PR TITLE
[doc] useful tools

### PR DESCRIPTION
**Pull request by plyfager**

### CODE REVIEW

File: docs/en/user_guides/useful_tools.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Useful tools
+
+We also provide tools for users.
```
Comment (LeoXing1996) [inline docs/en/user_guides/useful_tools.md:3]: In this tutorial, we introduce some useful tools for users.

File: docs/en/user_guides/useful_tools.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Useful tools
```
Comment (LeoXing1996) [inline docs/en/user_guides/useful_tools.md:1]: Tutorial 6: Useful tools

Rename this file to `6_useful_tools.md`

File: docs/en/user_guides/useful_tools.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+We also provide tools for users.
+
+## Print full config
+
+We incorporate modular and inheritance design into our [config](config.md) system, which is convenient to conduct various experiments. If you wish to inspect the config file, you may run `python tools/misc/print_config.py /PATH/TO/CONFIG` to see the complete config.
```
Comment (zengyh1900) [inline docs/en/user_guides/useful_tools.md:7]: + 'MMGeneration incorporates [config](config.md) mechanism to set parameters used for training and testing models. With our config mechanism, users can easily conduct extensive experiments without hard coding. If you wish to inspect ...........'

File: docs/en/user_guides/useful_tools.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+An Example:
+
+```python
+python tools/misc/print_config.py configs/styleganv2/stylegan2_c2-PL_8xb4-fp16-partial-GD-no-scaler-800kiters_ffhq-256x256.py
+```
+
+### Prepare a model for publishing
```
Comment (zengyh1900) [inline docs/en/user_guides/useful_tools.md:15]: '###'. --> '##'

File: docs/en/user_guides/useful_tools.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`tools/publish_model.py` helps users to prepare their model for publishing.
+
+Before you upload a model to AWS, you may want to
```
Comment (zengyh1900) [inline docs/en/user_guides/useful_tools.md:19]: move these steps before Line 17

**Pull request merged**